packages/eslint-plugin-pf-codemods/src/rules/v6/buttonMoveIconsIconProp/buttonMoveIconsIconPropInput.tsx

```diff
@@ -1,7 +1,18 @@
 import { Button } from "@patternfly/react-core";
```

Comment:
Related to the comment above, an `Icon` import should be included here in order for the second element to have the rule applied.